### PR TITLE
Homepage UI Revamp

### PR DESCRIPTION
**Pull request by ferrary7**
Changed the user interface of the homepage, added a search bar and promotion cards.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -78,7 +78,11 @@ body{
     border-radius: 5px;
 }
 
+
```
Comment (nikiljos) [inline style.css]: I think we can delete this comment...

Comment (ferrary7) [reply style.css]: Can you check this now ?

File: index.html [hunk 2]
```diff
@@ -7,8 +7,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Food Folio</title>
   <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/fd9a149f30.js" crossorigin="anonymous"></script>
```
Comment (nikiljos) [inline index.html]: Didn't notice anything from `fontawesome` being used in the code. Can u pls confirm if it is needed...

File: style.css (hunk 1 above)
Comment (nikiljos) [reply style.css]: 👍

File: index.html (hunk 2 above)
Comment (nikiljos) [reply index.html]: @ferrary7 Can u pls check

Comment (ferrary7) [reply index.html]: Done!

File: style.css [hunk 3]
```diff
@@ -1,4 +1,3 @@
-@import url('https://fonts.googleapis.com/css2?family=DM+Sans:wght@400;700&family=Poppins&display=swap');
```
Comment (nikiljos) [inline style.css:1]: This google fonts import cannot be removed! It's used in the stylesheet

File: index.html (hunk 2 above)
Comment (nikiljos) [reply index.html]: The script is still there right?

Comment (ferrary7) [reply index.html]: Yes

File: style.css (hunk 3 above)
Comment (ferrary7) [reply style.css:1]: Can we have a review now?

**Comment by nikiljos** [comment]
LGTM 👍

**Comment by rajuTK9** [comment]
LGTM

**Pull request merged**